src/platform_impl/web/web_sys/mod.rs

```diff
@@ -56,15 +56,19 @@ pub fn window_size() -> LogicalSize {
     LogicalSize { width, height }
 }
 
+pub fn hidpi_factor() -> f64 {
+    let window = web_sys::window().expect("Failed to obtain window");
+    window.device_pixel_ratio()
+}
+
 pub fn is_fullscreen(canvas: &HtmlCanvasElement) -> bool {
     let window = window().expect("Failed to obtain window");
     let document = window.document().expect("Failed to obtain document");
 
-    match document.fullscreen_element() {
-        Some(elem) => {
-            let raw: Element = canvas.clone().into();
-            raw == elem
```

Comment:
This line was causing a build error (that `Element` does not implement `PartialEq`).

Reply:
Huh, that's odd. I don't seem to have the same error?

Reply:
I'll get back to you with my specific versions when I can, but at the time of writing, I had a "clean" lockfile.